crates/air_r_formatter/src/lib.rs

```diff
@@ -213,7 +213,10 @@ where
 
     /// Returns `true` if the node is suppressed and should use the same formatting as in the source document.
     fn is_suppressed(&self, node: &N, f: &RFormatter) -> bool {
-        has_skip_comment(node.syntax(), f)
+        let node = node.syntax();
+        let comments = f.comments();
+        comments.mark_suppression_checked(node);
+        comments.has_skip_directive(node)
     }
```

Comment:
There is now this general pattern that your `is_suppressed()` overrides have to do two things:
- Call `mark_suppression_checked()`
- Call `has_skip_directive()`